backend/pyproject.toml

```diff
@@ -59,9 +52,16 @@ Tracker = "https://github.com/kitconcept/kitconcept.intranet/issues"
 [project.entry-points."z3c.autoinclude.plugin"]
 target = "plone"
 
+[tool.uv]
+managed = false
```

Comment:
I'm not sure if this is Ok, because you apparently do want UV to manage the environment and dependencies... It's missing from https://github.com/kitconcept/vmm-hoofdsite.project/blob/ca17ddd0b3f704a125107f9bdc2e4cfe1c45c3c5/backend/pyproject.toml

which is a setup where we tried to remove mxdev as well.

Reply:
Already commented on slack, uv is not managing dependencies in this setup. That's something we've been experimenting with as well.   let's talk at/after wrapup.